content/pages/tools/sphinx.html

```diff
@@ -0,0 +1,287 @@
+#}
+
+{% import "partials/basic.html" as basic %}
+{% import "partials/marketing.html" as marketing %}
+{% import "partials/technical.html" as technical %}
+{% import "partials/homepage.html" as homepage %}
+
+{% block signup_modal %}
+  {{ homepage.signup_modal("signup_modal") }}
+{% endblock signup_modal %}
+
+<section>
+  {% call basic.header_image(
+    header="Supercharge your Sphinx docs and impress your readers",
+    image="/images/readthedocs-thumb.png",
+    image_alt="Examples of Read the Docs hosted documentation",
+    vertical_align="middle")
+  %}
+  {# TODO: Add a GIF of using hoverxref & livesearch #}
+    <p>
+        Our powerful Sphinx hosting improves the reading and authoring experience for everyone who interacts with your documentation.
+    </p>
+
+    <p class="ui basic padded vertical segment">
+      <a class="ui large teal stackable button"
+      data-analytics="signup-modal"
+       onclick="jQuery('#signup_modal').modal('show');">
+        <i class="fad fa-play icon"></i>
+        Create an account
+      </a>
+      <a href="https://docs.readthedocs.io/en/stable/tutorial/index.html" class="ui large stackable button">
+        Start our tutorial
+      </a>
+    </p>
+  {% endcall %}
+</section>
+
+<section id="features">
+  {% call marketing.highlight_list() %}
+    {% call marketing.highlight_item(
+        icon="olive fa-people-arrows",
+        icon_style="--fa-secondary-color: violet; --fa-secondary-opacity: 1;",
+        header="Collaborate with deploy previews",
+        color="violet") %}
+    {% markdown %}
+    Receive feedback from your team,
+    and <strong>have confidence in changes</strong> before they go live.
+    {% endmarkdown %}
+    {% endcall %}
+
+    {% call marketing.highlight_item(
+        icon="yellow fa-magnifying-glass",
+        icon_style="--fa-secondary-color: white; --fa-secondary-opacity: 1;",
+        header="Always find what you need",
+        color="violet") %}
+    {% markdown %}
+    Search across all your projects with our search-as-you-type interface,
+    and quickly find the documentation and API reference information you need.
+    <strong>Never lose documentation again</strong>.
+    {% endmarkdown %}
+    {% endcall %}
+
+    {% call marketing.highlight_item(
+        icon="fa-eye",
+        header="Keep your focus with link previews",
+        color="violet") %}
+
+    {# TODO: Add link preview to the `link previews` docs #}
+
+    {% markdown %}
+    Provide context on every internal link in your docs with instant preview links.
+    <strong>Skim the content without opening *another* tab</strong>.
+    {% endmarkdown %}
+        {% endcall %}
+    {% endcall %}
+</section>
+
+<div class="ui container">
+  <div class="ui horizontal divider">Try it out</div>
+</div>
+
+<section id="try-it-out">
+  <div class="ui padded container">
+    <h2 class="ui huge center aligned header">
+      Upgrade your documentation with Read the Docs
```

Comment:
My original feedback is here: https://github.com/readthedocs/site-community/pull/152#discussion_r1015797061

I fixed the HTML structure, but might want to tweak this more.

Reply:
Yea, I don't love this heading. We were trying to address the idea that we do a lot _less_ than something like GH actions but also _more_ around docs, but that focus is valuable. I think that is still a strong selling point, and "upgrade your docs" doesn't really tell me much.

Reply:
> less than something like GH actions but also more around docs, but that focus is valuable

Aye, and that still seems like a worthwhile focus, but for different supporting content. The config file shows how to use RTD, but we haven't explained on this page that our focus is narrower than other services, or that it's a problem using a service that has too broad a focus.

We could/should totally have that content somewhere on our site though, but I'd expect the supporting content to describe why rolling your own solution is a bad idea, or show all the pieces you'd have to glue together to make RTD, etc.

Also, I'm very not tied to the heading I put up. The config example content is describing how easy it is to get benefits from RTD, so could use a heading that matches that sentiment.